lib/comgr/test/CMakeLists.txt

```diff
@@ -73,3 +74,8 @@ add_comgr_test(link_test)
 add_comgr_test(isa_name_parsing_test)
 add_comgr_test(get_data_isa_name_test)
 add_comgr_test(include_subdirectory_test)
+# Test : Compile HIP only if HIP-Clang is installed.
+find_package( hip CONFIG PATHS /opt/rocm )
+if (HIP_COMPILER AND ${HIP_COMPILER} STREQUAL "clang")
+  add_comgr_test(compile_hip_test)
```

Comment:
I think `test/compile_hip_test.c` is missing from the patch?

Reply:
will add it